src/renderingManager.js

```diff
@@ -300,6 +314,51 @@ export function newRenderingManager(win, environment) {
     }
   }
 
+  function registerMRAIDViewableEvent(callback) {
```

Comment:
this could be a performance concern based on how often the outer method is called. we should always avoid nested functions anyway as a best practice (this is part of our eslint rules that extend from eslint:recommended)